ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -14,10 +14,29 @@ import ReactiveCocoa
 class SignalSpec: QuickSpec {
 	override func spec() {
 		describe("init") {
-			pending("should run the generator immediately") {
-			}
-
-			pending("should keep signal alive if not terminated") {
+			let testError = NSError(domain: "SignalSpec", code: 1, userInfo: nil)
```

Comment:
Since signals can be parameterized by any `ErrorType`, I'd prefer if we could define a custom error `enum` for our unit tests.